README.md

```diff
@@ -176,33 +176,45 @@ Prerequisites
    - Docker Engine 1.17.6+ `docker version -f '{{.Server.APIVersion}}'`
    - Docker Compose 1.11.0+ `docker-compose --version`
 
-```sh
-go get github.com/google/keytransparency/...
```

Comment:
This failed with the below error message:
```sh
$ go get github.com/google/keytransparency/...

# github.com/google/keytransparency/impl/authorization/authz_go_proto
../google/keytransparency/impl/authorization/authz_go_proto/authz.pb.go:40:11: undefined: "github.com/golang/protobuf/proto".ProtoPackageIsVersion4
# github.com/google/keytransparency/core/sequencer/sequencer_go_proto
../google/keytransparency/core/sequencer/sequencer_go_proto/sequencer_api.pb.go:50:11: undefined: "github.com/golang/protobuf/proto".ProtoPackageIsVersion4
../google/keytransparency/core/sequencer/sequencer_go_proto/sequencer_api.pb.go:1205:7: undefined: grpc.ClientConnInterface
../google/keytransparency/core/sequencer/sequencer_go_proto/sequencer_api.pb.go:1209:11: undefined: grpc.SupportPackageIsVersion6
../google/keytransparency/core/sequencer/sequencer_go_proto/sequencer_api.pb.go:1234:5: undefined: grpc.ClientConnInterface
../google/keytransparency/core/sequencer/sequencer_go_proto/sequencer_api.pb.go:1237:43: undefined: grpc.ClientConnInterface
```
It seems to be related to some autogenerated file not committed in the repo.